src/components/InputField.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useFonts } from 'expo-font';
+import { Text, StyleSheet, Pressable, TextInput, View } from 'react-native';
+
+type Props = {
+  label: string;
+  isPassword: boolean;
+  onChange: (text: string) => void;
+};
+
+const InputField: React.FC<Props> = ({ label, isPassword, onChange }: Props) => {
```

Comment:
I strongly disagree with the idea of creating an InputField component abstracted away from the form that contains it. This should be its own PR at the very least. It also makes it harder to implement input validation(eg passwords of certain length/characters, or valid email addresses/valid names etc)

Reply:
I had the button component added to the Welcome Screen PR, and it was accepted. So thought this was okay. Will keep it more organized in the future in separate PR's. Please accept the Input Field along with the third party button component in this PR this time, as it is hard and confusing to separate them to different PR's now. Will keep PR's small and separate in the future.

Reply:
That will be up to @MelisaIm after she reviews this code. This code is not adequately tested, render tests are just for example. This is also true for the other components. I personally am reluctant to add so many untested components all at once.

Reply:
Writing testing files is something I am new to and would love to spend some time learning and working on them.